src/fontstash.h

```diff
@@ -1380,7 +1410,7 @@ int fonsTextIterNext(FONScontext* stash, FONStextIter* iter, FONSquad* quad)
 		// Get glyph and quad
 		iter->x = iter->nextx;
 		iter->y = iter->nexty;
-		glyph = fons__getGlyph(stash, iter->font, iter->codepoint, iter->isize, iter->iblur);
+		glyph = fons__getGlyph(stash, iter->font, iter->codepoint, iter->isize, iter->iblur, FONS_GLYPH_BITMAP_REQUIRED);
```

Comment:
There should be an iterator which calls getGlyph using optional. That way `nvgTextGlyphPositions` and `nvgTextBreakLines`, `nvgTextBoxBounds` will not render glyphs either. 

Maybe the best way would be to add additional parameter to `fonsTextIterInit`. The `fonsTextIterNext ` function declaration should have comment that if the iterator was initialized with optional, then the UV coordinates of the quad will be invalid.